### DISCUSSION TITLE
which column to use for mokapot generated rescoring results?

### DISCUSSION THREAD

**Discussion by frankligy** [CompOmics/ms2rescore#196] [Q&A] [answered]
Hello,

Sorry I have another question, so for the mokapot generated results, I usually look for the `ms2rescore.mokapot.psms.txt`, based on my understanding of the output, I was planning to use the column called "mokapot q-value" such that If I use FDR 5%, then I should retain ones with this column less than 0.05, because that's the definition of q-value right?

But I also saw the mokapot score and seems the generated figure cut off at 0, and there's also a PEP column, so just want to confirm with you how would you interpret the mokapot result and to filter based on a defined FDR.

Thank you,
Frank

**Comment by RalfG** [comment]
Hi Frank,

Thanks for reaching out! The `*.mokapot.psms.txt` contains all target peptide-spectrum matches (PSMs), so you can indeed use the `mokapot q-value` to filter to any FDR level. Keeping only PSMs with a q-value < 0.05, will indeed result in an estimated FDR-level of 5%. Do note that we usually recommend filtering at 1% or lower.

For Mokapot (and Percolator), the `score` value is the direct output of the learned model, where PSMs with a value lower than 0 are predicted to be false, and PSMs with a value higher than 0 to be true (see https://doi.org/10.1021/acs.jproteome.0c01010). It can be useful for plotting score distributions, but for most downstream analysis, the q-value and PEP are all you need.

There is a great paper describing both the q-value and the PEP by Lukas Käll et al: https://doi.org/10.1021/pr700739d. Very briefly, the q-value says something about the dataset as a whole, while the PEP says something about a single PSM. In your case, for filtering a dataset to a given FDR-level, the q-values should be used.

Best,
Ralf

**Reply by frankligy** [reply, marked as answer]
Thanks a lot for the explanations!
